rbc/heavydb/remoteheavydb.py

```diff
@@ -1266,8 +1268,35 @@ def _make_udf(self, caller, orig_sig, sig):
             name + sig.mangling(),
             atypes, rtype, annotations)
 
+    def _check_llvm_version(self):
+        # check LLVM version before compiling to LLVM
+        flag_name = 'RBC_DISABLE_LLVM_MISMATCH_WARN'
+        flag = int(os.environ.get(flag_name, False))
+
```

Comment:
We should check the LLVM version and possibly warn only once. One possible approach is as follows:
```suggestion
        os.environ[flag_name] = 1
```